README.md

```diff
@@ -217,6 +217,24 @@ Stop the tunnel with: `make <env> stop-tunnel`
 ## Other useful commands
 Type `make` to get the list of all available commands.
 
+## Known caveats
+
+### Fly version
+
+Current mechanism of updating fly binary only checks if the date of your local
+binary is older than one on the server. If you have been using previous fly
+version recently, it won't get updated. In that case, delete the binary in /bin
```

Comment:
For example when you are upgrading from 0.7 (previous version). If you have downloaded it recently, curl won't update it and you get some cryptic error like `Resource type not found`.